packages/swc-configs/CHANGES.md

```diff
@@ -0,0 +1,27 @@
+- Each config exports it's defaults
```

Comment:
whats the purpose of this file? should it be removed?

Reply:
Deleted